packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -217,7 +213,10 @@ export class QuoteRequestor {
         options: RfqtRequestOpts,
     ): Promise<RFQTIndicativeQuote[]> {
         const _opts: RfqtRequestOpts = { ...constants.DEFAULT_RFQT_REQUEST_OPTS, ...options };
-        assertTakerAddressOrThrow(_opts.takerAddress);
+
+        if (!_opts.takerAddress) {
```

Comment:
Would you mind adding a quick comment explaining why we set indicative quote `takerAddress` to `NULL_ADDRESS`?

Reply:
Addressed in https://github.com/0xProject/0x-monorepo/pull/2684/commits/a8f3a47240c1a0adabbdd60ca82c8c98b4f07851